clouds/cli/cli.py

```diff
@@ -0,0 +1,106 @@
+import os
+import click
+import shutil
+from simple_logger.logger import get_logger
+from clouds.cli.aws.aws_cli import clean_aws_resources
+from pyhelper_utils.runners import function_runner_with_pdb
+from clouds.cli.microsoft_azure.microsoft_azure_cli import nuke_all_azure_resources
+from clouds.aws.aws_utils import set_and_verify_aws_credentials, aws_region_names
+
+LOGGER = get_logger(name="nuke-cli")
+
+
+@click.group()
+def aws() -> None:
+    pass
+
+
+@aws.command()
+@click.option(
+    "--aws-regions",
+    help="""
+        \b
+        Comma-separated string of AWS regions to delete resources from.
+        If not provided will iterate over all regions.
+        """,
+    required=False,
+)
+@click.option(
+    "--all-aws-regions",
+    help="""
+        \b
+        Run on all AWS regions.
+        """,
+    is_flag=True,
+)
+def aws_nuke(aws_regions: str, all_aws_regions: bool) -> None:
+    """
+    Nuke all AWS cloud resources in given/all regions
+    """
+
+    if all_aws_regions:
+        _aws_regions = aws_region_names()
+    elif aws_regions:
+        _aws_regions = aws_regions.split(",")
+    else:
+        click.echo("Either pass --all-aws-regions or --aws-regions to run cleanup")
+        raise click.Abort()
```

Comment:
`click.Abort()` not fits here, also it's harder to tests it.
Please create exceptions to match your errors.

For example:
```
class UserInputError(Exception):
    pass
```

Reply:
done